src/apps/relay/mainrelay.c

```diff
@@ -1345,7 +1345,6 @@ static void set_option(int c, char *value)
 		STRCPY(turn_params.relay_ifname, value);
 		break;
 	case 'm':
-#if defined(OPENSSL_THREADS)
```

Comment:
Slowly chipping away compatibility with super-old openssl versions
Running turnserver in a single threaded mode using very old openssl versions is hard to support
Worst case we can branch away and support 4.6.x versions